be/test/storage/lake/primary_key_compaction_task_test.cpp

```diff
@@ -65,6 +65,7 @@ class LakePrimaryKeyCompactionTest : public TestBase, public testing::WithParamI
         CHECK_OK(fs::create_directories(lake::join_path(kTestDirectory, lake::kMetadataDirectoryName)));
         CHECK_OK(fs::create_directories(lake::join_path(kTestDirectory, lake::kTxnLogDirectoryName)));
         CHECK_OK(_tablet_mgr->put_tablet_metadata(*_tablet_metadata));
+        config::lake_pk_compaction_min_input_segments = 2;
```

Comment:
remember to reset this configvar to its default value.

Reply:
Ok

Reply:
Done